app/javascript/components/AddItem.js

```diff
@@ -1,5 +1,75 @@
-import React from 'react';
+import React, { useState } from 'react';
+import axios from 'axios';
 
-const AddItem = () => <div>AddItem</div>;
+const AddItem = () => {
+  const [carData, setCarData] = useState({
+    name: '',
+    description: '',
+    finance_fee: '',
+    purchase_fee: '',
+    total_amount: '',
+    apr: '',
+    snapshot: null,
+  });
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    axios.post('http://localhost:3000/api/v1/cars', carData)
+      .then((response) => {
+        console.log('Car added successfully:', response.data);
+      })
+      .catch((error) => {
+        console.error('Error adding car:', error);
+      });
+  };
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setCarData((prevData) => ({
+      ...prevData,
+      [name]: value,
+    }));
+  };
+  
+  return (
+    <div className="addItem_container">
+      <form onSubmit={handleSubmit}>
+        <h3 className="add-form-title">Add an Item</h3>
+        <div className="form-group">
+          <label htmlFor="name">
+           
+            <input type="text" name="name" value={carData.name} onChange={handleChange} required placeholder="Name"/>
+          </label>
+          <label htmlFor="description">
+            
+            <input type="text" name="description" value={carData.description} onChange={handleChange} required placeholder="Description"/>
+          </label>
+          <label htmlFor="finance_fee">
+          
```

Comment:
Use console logs to check your inputs are being sent correctly cause in the object creation they are getting the default value as we seen on your presentation